robottelo/ui/base.py

```diff
@@ -463,3 +420,24 @@ def is_element_visible(self, locator):
             return False
         self.wait_for_ajax()
         return element.is_displayed()
+
+    def click(self, locator, wait_for_ajax=True, timeout=30):
```

Comment:
I think that does not make sense now as the timeout for the wait_until_element will not be different from the default one. If that is necessary we can add that with the refactor of the other methods.